base/special/log.jl

```diff
@@ -397,51 +397,167 @@ function log1p(x::Float32)
     end
 end
 
-
-@inline function log_ext_kernel(x_hi::Float64, x_lo::Float64)
-    c1hi = 0.666666666666666629659233
-    hi_order =  evalpoly(x_hi, (0.400000000000000077715612, 0.285714285714249172087875,
-                                0.222222222230083560345903, 0.181818180850050775676507,
-                                0.153846227114512262845736, 0.13332981086846273921509,
-                                0.117754809412463995466069, 0.103239680901072952701192,
-                                0.116255524079935043668677))
-    res_hi, res_lo = two_mul(hi_order, x_hi)
-    res_lo = fma(x_lo, hi_order, res_lo)
-    ans_hi = c1hi + res_hi
-    ans_lo = ((c1hi - ans_hi) + res_hi) + (res_lo + 3.80554962542412056336616e-17)
-    return ans_hi, ans_lo
-end
+const t_log_ext_Float64 = (
+(0x1.6a00000000000p+0, -0x1.62c82f2b9c800p-2, 0x1.ab42428375680p-48),
```

Comment:
The first and second entries of this table can be combined into one entry. Worth thinking about what the best way to do that given available instructions, but it'd save 1/3 the table size and prevent values from the same row from straddeling a cache line, which is probably worth it.

Reply:
that's probably smart. The way to combine them is to mask off `0x3f` from the first entry and shift it into the bottom of the second entry. Undoing this takes 3 instruction (2 ors and 1 shift)

Reply:
one pshufb is probably better, but that's kinda llvm's job to figure out.